docs/src/stDDM.md

```diff
@@ -0,0 +1,106 @@
+# Starting-time Drift Diffusion Model (stDDM)
+
+The relative starting time drift diffusion model (stDDM) characterizes the contributions of multiple unique attributes to the rate of evidence accumulation. Compared to the DDM, which assumes a constant evidence accumulation rate within each trial, the stDDM allows different attributes to enter the evidence accumulation process at various time points relative to one another. By doing so, the stDDM quantifies both the weights given to each attribute and their onset times (Amasino et al., 2019; Barakchian et al., 2021; Chen et al., 2022; Maier et al., 2020; Sullivan and Huettel, 2021).
+
+# Example
+In this example, we will demonstrate how to use the stDDM in a generic two-alternative forced-choice task with two arbitrary attributes.
+
+## Load Packages
+The first step is to load the required packages.
+
+```@example stDDM
+using SequentialSamplingModels
+using Plots
+using Random
+
+Random.seed!(8741)
+```
+
+## Create Model Object
+In the code below, we will define parameters for the stDDM and create a model object to store the parameter values. 
+
+### Drift Rates
+The drift rates control the speed and direction with which information accumulates, with one drift rate per attribute (e.g., taste and health, payoff and delay, self and other).
```

Comment:
If possible, can you briefly explain how the model processes multiple alternatives if the drift rates correspond to attributes.

Reply:
Maybe the winner depends on which alternative is attended first?